inc/namespace.php

```diff
@@ -30,6 +29,12 @@ function bootstrap() {
 		return 'optin';
```

Comment:
This is duplicating https://github.com/humanmade/consent-api-js/blob/main/inc/namespace.php#L56-L61 (except I'm `__doing_it_wrong` over there 🤦‍♂️) so this can probably be removed (after it's fixed in our consent-api fork).

Reply:
Basically, part of the way a plugin integrates into the consent api is to use the `wp_consent_type` filter to override the default consent type (if desired). In the base plugin, they default to `optout` so we'd want to override, but in our fork we default to `optin` (when I'm doing it right...).

Reply:
Ha! Gnarly, thanks for the explainer